openssf-compiler-options.yaml

```diff
@@ -20,6 +20,20 @@ pipeline:
       for i in $(seq 15 18); do
         ln -s clang "${{targets.destdir}}"/etc/clang-$i
       done
+      cd ${{targets.destdir}}/usr/local/bin
+      ln -s gcc-wrapper cc
+      ln -s gcc-wrapper cpp
+      for suffix in `seq 12 14`; do
```

Comment:
can you call this 'version' ? i think that is what it is, or 'major'.
and then just drop 'seq' and use 12 13 14
```patch
- for suffix in `seq 12 14`; do
+ for major in 12 13 14; do
```

It is more "greppable" and also shorter.